gax-java/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -91,7 +98,13 @@ public abstract class RetrySettings implements Serializable {
    * Duration.ZERO} and LROs have a default total timeout value of {@code Duration.ofMillis(300000)}
    * (5 minutes).
    */
-  public abstract Duration getTotalTimeout();
+  public final java.time.Duration getTotalTimeoutDuration() {
```

Comment:
Should this be final?

Reply:
As per https://github.com/googleapis/sdk-platform-java/pull/1872#discussion_r1619231111, we use the keyword to ensure the behavior mirrors the `threeten` counterpart that has been implemented